### PR TITLE
via: Report if failed test is due failure or if test is missing

### PR DESCRIPTION
**Pull request by davidlunarg**
Addresses issue https://github.com/LunarG/VulkanTools/issues/948

### CODE REVIEW

File: via/via_system_linux.cpp [hunk 1]
```diff
@@ -89,6 +90,8 @@ int ViaSystemLinux::RunTestInDirectory(std::string path, std::string test, std::
             err = chdir(orig_dir);
         }
     }
+    if (err_code && err_code != 1)
```
Comment (MarkY-LunarG) [inline via/via_system_linux.cpp:93]: Nit: Wrap conditional code in braces.

File: via/via_system.cpp [hunk 2]
```diff
@@ -2052,11 +2051,6 @@ ViaSystem::ViaResults ViaSystem::GenerateTestInfo(void) {
             full_cmd = cube_exe;
             full_cmd += " --c 100 --suppress_popups";
             int test_result = RunTestInDirectory(path, cube_exe, full_cmd);
-            if (test_result == 0) {
```
Comment (MarkY-LunarG) [inline via/via_system.cpp:2055]: Removing this will make it attempt both times to run cube, why not just re-add the if (test_result != 0) { continue; }?

**Review by MarkY-LunarG — CHANGES_REQUESTED** [review]

File: via/via_system.cpp (hunk 2 above)
Comment (davidlunarg) [reply via/via_system.cpp:2055]: If the test failed, we want to output some indication of failure. The code in the loop after the continue does that.

Also, there's a PrintEndTable that is always executed after the loop is exited. If we do a continue here, the corresponding PrintBeginTable is not executed.

File: via/via_system_linux.cpp (hunk 1 above)
Comment (davidlunarg) [reply via/via_system_linux.cpp]: Commit 71c075d9e59071ceffffe55b6a56bf18c24a58ad addressed this.

**Review by MarkY-LunarG — APPROVED** [review]

**Pull request merged**